stepper-2.0.md

```diff
@@ -40,7 +40,7 @@ Protocol
 6  num steps-per-revolution LSB
 7  num steps-per-revolution MSB
 8  maxSpeed LSB
```

Comment:
We can probably get away with maxSpeed as a 14-bit number if we don't require a multiplier for it like we do when setting the speed value. In this case maxSpeed would probably rarely exceed 1000.